app/swagger/schemas/gibct/institutions.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+module Swagger
+  module Schemas
+    module Gibct
+      class Institutions
+        include Swagger::Blocks
+
+        swagger_schema :GibctInstitutionsAutocomplete do
+          key :required, [:data, :meta, :links]
+
+          property :data, type: :array, minItems: 1, uniqueItems: true do
```

Comment:
minItems should be 0, right? Autocomplete for "zyv" will return an empty array, for example.